be1-go/channel/reaction/mod_test.go

```diff
@@ -500,8 +500,8 @@ func Test_DeleteReaction_Out_of_Order(t *testing.T) {
 
 	// Publishing the reaction with some delay in a go routine
 	go func() {
-		time.Sleep(50 * time.Millisecond)
-		err = cha.Publish(pub, socket.ClientSocket{})
+		time.Sleep(time.Millisecond * 50)
```

Comment:
I'd try and keep `50 * time.Millisecond` as it reads more like natural language (you'd say 50 seconds, not seconds 50)